### PR TITLE
enhance: clean the local-persist after integration tests

### PR DESCRIPTION
**Pull request by shaloulcy**
Signed-off-by: Eric Li <lcy041536@gmail.com>

### Ⅰ. Describe what this PR did

After integration tests, we will clean the local-persist volume plugin

### Ⅱ. Does this pull request fix one issue?

#1396

### CODE REVIEW

File: hack/make.sh [hunk 1]
```diff
@@ -117,6 +117,21 @@ function install_local_persist
 	mv /tmp/local-persist /usr/local/bin/
 }
 
+# clean the local-persist
+function clean_local_persist
+{
+	echo "Try cleaning local-persist"
+	pid=`ps axu|grep local-persist |grep -v grep|awk '{print $2}'`
```
Comment (Letty5411) [inline hack/make.sh]: How about just pgrep ?

File: hack/make.sh [hunk 2]
```diff
@@ -117,6 +117,21 @@ function install_local_persist
...
+
+	if [ -n "$pid" ]; then
```
Comment (Letty5411) [inline hack/make.sh]: use [[ ]] instead of [ ]

File: hack/make.sh (hunk 1 above)
Comment (shaloulcy) [reply hack/make.sh]: done it

File: hack/make.sh (hunk 2 above)
Comment (shaloulcy) [reply hack/make.sh]: done it

**Comment by rudyfly** [comment]
LGTM

**Pull request merged**